app-modules/authorization/src/Filament/Pages/Auth/Login.php

```diff
@@ -241,8 +258,10 @@ protected function getForms(): array
                     ->schema([
                         $this->getEmailFormComponent()
```

Comment:
I think the label here needs to be changed to "Email" rather than what it currently is "Email address"